apps/dav/lib/Migration/BuildSocialSearchIndexBackgroundJob.php

```diff
@@ -71,7 +71,11 @@ private function buildIndex($offset, $stopAt) {
 		// refresh identified contacts in order to re-index
 		foreach ($social_cards as $contact) {
 			$offset = $contact['id'];
-			$this->davBackend->updateCard($contact['addressbookid'], $contact['uri'], $contact['carddata']);
+			$cardData = $contact['carddata'];
+			if (is_resource($cardData) && (get_resource_type($cardData) == 'stream')) {
```

Comment:
```suggestion
			if (is_resource($cardData) && (get_resource_type($cardData) === 'stream')) {
```

Learn more at https://docs.nextcloud.com/server/latest/developer_manual/getting_started/codingguidelines.html#operators